From 6b38a8a6d5bfe201305274bd58e0fc34fc383d76 Mon Sep 17 00:00:00 2001
From: John Crispin <john@phrozen.org>
Date: Wed, 17 Jan 2024 09:58:18 +0100
Subject: [PATCH 27/55] ramips: reduce yuncore ax820 SPI clock speed to 20MHz

Reduces SPI flash clock speed to improve stability and prevent read errors on this device.

Signed-off-by: John Crispin <john@phrozen.org>
---
 target/linux/ramips/dts/mt7621_yuncore_ax820.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/target/linux/ramips/dts/mt7621_yuncore_ax820.dts b/target/linux/ramips/dts/mt7621_yuncore_ax820.dts
index 51421aba6a..3bdcfe439c 100644
--- a/target/linux/ramips/dts/mt7621_yuncore_ax820.dts
+++ b/target/linux/ramips/dts/mt7621_yuncore_ax820.dts
@@ -87,7 +87,7 @@
 	flash@0 {
 		compatible = "jedec,spi-nor";
 		reg = <0>;
-		spi-max-frequency = <50000000>;
+		spi-max-frequency = <20000000>;
 		m25p,fast-read;
 
 		partitions {
-- 
2.34.1

